internal/exec/stages/umount/umount.go

```diff
@@ -80,7 +80,7 @@ func (s stage) Run(config types.Config) error {
 }
 
 func (s stage) umountFs(fs types.Filesystem) error {
-	if fs.Format != nil && *fs.Format == "swap" {
+	if fs.Format != nil && (*fs.Format == "swap" || *fs.Format == "none") {
```

Comment:
Not new to this PR, though feels like this should also check `fs.Format == ""` to be symmetric to the mount stage.

Reply:
Yeah, makes sense